library/database/class.database.php

```diff
@@ -70,6 +70,15 @@ class Gdn_Database {
     /** @var int Number of retries when the db has gone away. */
     public $ConnectRetries;
 
+    /** @var int Number of milliseconds we are willing to wait to make a successful connection to the DB */
+    protected $SmoothTimeoutMillis;
+
+    /** @var int Minimum number of milliseconds we want to wait before attempting a new DB connection*/
+    protected $SmoothWaitMinMillis;
+
+    /** @var int Maximum number of milliseconds we want to wait before attempting a new DB connection*/
+    protected $SmoothWaitMaxMillis;
+
```

Comment:
I feel we should be using the new coding standards for new code, even inside old files that don't currently conform. What do you think?

Reply:
@egarciajulia What _do_ you think?

Reply:
I wouldn't touch it at this point.
I agree that we could improve/refactor the class and apply the new standard, however I would do it as a different PR to avoid any unnecessary side effects.